catalog/format/iceberg/src/main/java/org/projectnessie/catalog/formats/iceberg/nessie/NessieModelIceberg.java

```diff
@@ -795,14 +793,8 @@ public static long safeUnbox(Long value, long defaultValue) {
    *
    * <p>Also: we deliberately ignore the TableProperties.WRITE_METADATA_LOCATION property here.
    */
-  public static String icebergBaseLocation(String warehouseLocation, ContentKey key) {
+  public static String icebergNewEntityBaseLocation(String baseLocation) {
     // FIXME escape or remove forbidden chars, cf. #8524
-    String baseLocation = warehouseLocation;
-    Namespace ns = key.getNamespace();
-    if (!ns.isEmpty()) {
-      baseLocation = concatLocation(warehouseLocation, ns.toString());
```

Comment:
The new base location construction logic seems to be different from the old... Shall we add a CHANGELOG notice?

Reply:
How's it different?

Reply:
`ns.toString()` uses `.` for namespace separators, `.resolve()` (in the other comment) user `/`.

Reply:
heh - then the old one was kinda incorrect.

Reply:
Incorrect of not, it's a behaviour change... I think it's worth a CHANGELOG entry.